CITATION

```diff
@@ -0,0 +1,14 @@
+To cite PROJ in publications use:
+
+  PROJ contributors (2018). The PROJ coordinate transformation software
+  library. The Open Source Geospatial Foundation. URL http://proj4.org/.
+
+A BibTeX entry for LaTeX users is
+
+  @Manual{,
+    title = {The {PROJ} coordinate transformation software library},
+    author = {{PROJ contributors}},
+    organization = {The Open Source Geospatial Foundation},
+    year = {2018},
+    url = {http://proj4.org/},
+  }
```

Comment:
There's a nasty special character at the end of the file here.

Reply:
Ah was rather the lack of newline character. Fixed.